src/tools/immutable_object_cache/ObjectCacheStore.h

```diff
@@ -30,30 +32,46 @@ class ObjectCacheStore {
   int init_cache();
   int lookup_object(std::string pool_nspace,
                     uint64_t pool_id, uint64_t snap_id,
-                    std::string object_name,
+                    uint64_t object_size, std::string object_name,
                     std::string& target_cache_file_path);
 
  private:
+  enum ThrottleTypeCode {
+    THROTTLE_CODE_BYTE,
+    THROTTLE_CODE_OBJECT
+  };
+  #define ROC_QOS_IOPS_THROTTLE 1 << 0
```

Comment:
Nit: avoid preprocessor definitions -- use an enum instead (like the one above).